OneCommunityGlobal/HGNRest
# Yihan send review email

# Description
<img width="735" alt="截屏2023-07-25 19 15 37" src="https://github.com/OneCommunityGlobal/HGNRest/assets/94303659/d6cc6a3a-bb3b-4489-a078-d3e400bdd68c">


## Related PRS (if any):
To test this backend PR you need to checkout the [#1045](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1045) frontend PR.
This PR is based on [PR#443](https://github.com/OneCommunityGlobal/HGNRest/pull/443). I recommand to test #443 before review this one.

## Main changes explained:
- Update file taskController.js and taskRouter.js for adding send email function.

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. follow the steps in the test instruction of [PR#1045](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1045).
